oap-ws-sso-api/pom.xml

```diff
@@ -55,6 +55,11 @@
             <artifactId>guava</artifactId>
             <version>${oap.deps.guava.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.auth0</groupId>
+            <artifactId>java-jwt</artifactId>
+            <version>3.19.2</version>
```

Comment:
move to properties  please
oap.deps.jwt.version

Reply:
agreee